helm/starboard-exporter/values.yaml

```diff
@@ -56,13 +56,21 @@ exporter:
     enabled: true
 
   vulnerabilityReports:
-    enabled: true
+    # Shared configuration for all vulnerability scanners
     targetLabels: []
       # - image_namespace
       # - image_repository
       # - image_tag
       # - vulnerability_id
 
+    # Scanner-specific configuration
+    # Possible Label Names: scanner / report_producer / report_source / something else?
```

Comment:
That terminology discussion was for the label
```suggestion
```

But as I think about it, when we "finish" this values interface, it probably makes more sense to focus on the resource type and not on the tool name. Multiple tools could theoretically produce reports of the same type. So this could be something more like:
```
vulnerabilityReports:
  targetReportTypes:
    - vulnerabilitymanifests.etc.etc
    - vulnerabilityreports.aqua.etc
```

I'll think about the second part with the team, but let's take out the comment still